### DISCUSSION TITLE
Pico theme development design ideas

### DISCUSSION THREAD

**Discussion by ratulkhan74** [cadentcom/pico-wp-block-theme#8] [Ideas]
**_Design ideas link:_**

Idea from Neil: 

https://www.wix.com/website-template/view/html/1820?originUrl=https%3A%2F%2Fwww.wix.com%2Fwebsite%2Ftemplates%3Fcriteria%3Dhealthcare&tpClick=view_button&esi=66c033f9-2a95-466d-b9b8-b58445c8b7cf

Design ideas research by Ratul:

**Webflow templates:**

https://medicio.webflow.io/home/home-one (good one)
https://medicio.webflow.io/home/home-two
https://medicatemplate.webflow.io/home
https://diagonsy-template.webflow.io/ (I like)
https://diagonsy-template.webflow.io/home-2
https://clinic-medical-template.webflow.io/
https://wellcare-template.webflow.io/ (good one)
https://doctorate-template.webflow.io/
https://mediflow-template.webflow.io/ (I like)
https://marseille-template.webflow.io/ (I like)
https://doctortemplate.webflow.io/home

**Template monster:**

https://www.iideainformatics.it/html/medicaid/index.html (good one)
https://barakahit.net/html/medicare/index.html (I like)
https://demo.themexbd.com/html/medeno/index.html

**Comment by stratofax** [comment]
@ratulkhan74 Here's my review of the themes listed above.

## Wix

The Wix theme at [Medical Clinic Website Template | WIX](https://www.wix.com/website-template/view/html/1820?originUrl=https%3A%2F%2Fwww.wix.com%2Fwebsite%2Ftemplates%3Fcriteria%3Dhealthcare&tpClick=view_button&esi=66c033f9-2a95-466d-b9b8-b58445c8b7cf) is incomplete, because it's missing some sections on the home page.

### What's Good

- The color palette is the best we've seen so far.
- The three-layer header looks good and is flexible for different menus and layouts. See below
- The hero image is a full-width photo, this is the best hero image layout (not image subject) in the bunch.
- The purple CTA strip below the full-width hero image is excellent
- The overlay "tabs" at the bottom are a good idea, but may be optional features

### What's bad

- Putting the address and phone number (contact info) on the middle strip in the header is not a good look. Better to use this space for a secondary menu (Search, socials, Donate, Translate, etc).
- The font, `raleway`, is too light and thus reduces readability and accessibility
- The hero slider should be removed, for a single still image, for best useability.
- The three-shades-of green boxes overlaid on the hero make the section too busy. Maybe leave one for a call to action (CTA)?

## Webflow

- [Home - Doc - Webflow Ecommerce website template](https://doctortemplate.webflow.io/home) uses a warm green-and-orange palette. Good font selection: DM Serif for headers, DM Sans for body text. Google fonts?
- [Doctorate - Webflow HTML website template](https://doctorate-template.webflow.io/)($49 USD) includes a good services section. Fonts: Poppins for headings, DM Sans for body copy. _Cold_ color palette.
- [Medicio - Webflow HTML Website Template](https://medicio.webflow.io/home/home-one) uses a great font, **Outfit**: Semi-Bold for headings, Regular for body copy. Is this a Google font? Childish design elements suggest pediatrics. The "cut-out" person in the hero image is no good, too brittle for different layouts.
- Variant [Home Two](https://medicio.webflow.io/home/home-two) includes an OK color palette, a good layout for the **Why Choose Us** section. We don't like the "slide in" effect for page elements, or the things that move for no reason.

The other themes were not worth emulating.

## Next Steps

### Colors and Fonts

@ratulkhan74 please focus on these tasks:

- [x] Create a color palette specification based on the Wix theme at [Medical Clinic Website Template | WIX](https://www.wix.com/website-template/view/html/1820?originUrl=https%3A%2F%2Fwww.wix.com%2Fwebsite%2Ftemplates%3Fcriteria%3Dhealthcare&tpClick=view_button&esi=66c033f9-2a95-466d-b9b8-b58445c8b7cf)
- [x] Select fonts from Webflow samples, if available from Google, or find equivalents.
- [x] Finalize colors and fonts
- [x] Update the `theme.json` file with these selections
- [x] Review & PR

### Homepage Sections

@stratofax will peform these tasks:

- [ ] Create a wireframe for the sections to implement in version 1 of the homepage.
- [ ] Post for review

**Comment by ratulkhan74** [comment]
@stratofax,  Pico theme has been created and listed tasks are done

**Comment by stratofax** [comment]
Since this discussion is primarily concerned with another theme, I'm going to close it.
